.github/workflows/manifest.yml

```diff
@@ -89,4 +89,5 @@ jobs:
       - name: Repository Dispatch
         uses: peter-evans/repository-dispatch@v2
         with:
-          event-type: build-firmware
\ No newline at end of file
+          event-type: build-firmware
+          client-payload: '{"tag": "${{ steps.tag.outputs.latestTag }}"}'
```

Comment:
Is this still valid?

Reply:
Yes, that is the value that is being pulled by this file and pushed over to the build file. I guess I could just combine the two files if you want?

Reply:
I have merged the files into one release workflow action. So no need for this anymore.